digitalocean/resource_digitalocean_kubernetes_cluster.go

```diff
@@ -367,26 +367,16 @@ func flattenKubeConfig(config *godo.KubernetesClusterConfig) []interface{} {
 }
 
 // we need to filter tags to remove any automatically added to avoid state problems,
-// these are tags starting with "k8s:", named "k8s" or duplicates of the cluster tags
-func filterTags(tags []string, parentTags ...string) []string {
+// these are tags starting with "k8s:" or named "k8s"
+func filterTags(tags []string) []string {
```

Comment:
I'd normally have test cases for the different conditions here but if you don't thing it's necessary that's fine with me.

Reply:
:+1:  Yup, should have gotten a test when first added. We should be more consistent in making sure that unit tests are added in addition to the integration tests. Pushed a new commit with a test.